src/LodeRunner.API/src/Handlers/ResultHandler.cs

```diff
@@ -62,25 +69,28 @@ public static JsonResult CreateCancellationInProgressResult()
         /// <param name="errorList">list of validation errors.</param>
         /// <param name="path">string.</param>
         /// <returns>JsonResult.</returns>
-        public static JsonResult CreateResult(List<ValidationError> errorList, string path)
+        public static async Task<JsonResult> CreateResult(List<ValidationError> errorList, string path)
```

Comment:
Same here.  It looks like we have 3 overloads, but necessarily since they are called differently and they have slightly different purposes.  If we can make 2 of them call to one then the overload makes sense, but if they are just different then let's name them differently.

Reply:
good suggestion, done.